### DISCUSSION TITLE
Independent outdoor unit

### DISCUSSION THREAD

**Discussion by Czbricht** [issalig/toshiba_air_cond#16] [Q&A]
Hi, I want to use only outdoor unit as air /water. Do you think that only the outdoor unit can be turned on and used this way? Maybe only in test mode if it can be turned on permanently at maximum power?

**Comment by issalig** [comment]
Sorry, I have no idea, mine is only air. If you discover sth just let me know.

**Comment by stapad** [comment]
[issalig](https://github.com/issalig) I would like to thank you for your work, it's inspiration for myself to move one step further. Namely, I would like to remove entirely internal unit and use external unit as a air/water heat pump. There is are 3 wires wide connection between internal & external unit where one of them is UART 1-Wire communication. Have you got a chance to sniff that interface too?
